onnxruntime/test/providers/cpu/tensor/resize_op_test.cc

```diff
@@ -532,7 +534,8 @@ TEST(ResizeOpTest, NhwcResizeOpLinearDownSampleTest_4DBilinear_pytorch_half_pixe
   test.AddOutput<uint8_t>("Y", {N, sizes[1], sizes[2], C}, Y);
   // CUDA: result mismatch due to not implementing NHWC support
   // ROCm: results mismatch
-  test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kCudaExecutionProvider, kRocmExecutionProvider});
+  // DML: results mismatch
```

Comment:
Hmm, that might be due to downsampling by a large size. I see we don't use RESAMPLE3 yet in core\providers\dml\DmlExecutionProvider\src\Operators\DmlOperatorResize.cpp.

Reply:
Yeah, I think this PR is targeting align with DML 6.2, @linnealovespie will take the task to use RESAMPLE3 and leverage the enabled tests I believe when DML 1.13.1 is released.